cypress/integration/article-view-spec.js

```diff
@@ -56,12 +56,9 @@ describe('Article view', () => {
     goToCatArticle()
     articlePage.selectOptionFromActionsMenu('gallery')
     articlePage.galleryImage().should('be.visible')
-    articlePage.galleryImage().invoke('attr', 'src').then((src) => {
-      cy.rightArrow()
-      articlePage.galleryImage().should('not.have.attr', 'src', src)
-    })
     cy.enter()
     articlePage.galleryPopupHeader().should('be.visible')
+    articlePage.galleryPopupDescriptionText().should('be.visible').should('have.text', 'A montage of cat pictures using images from Wikimedia creators')
```

Comment:
@medied can u point me to the call/url/api we use to get this information about the image?
this test passes now but if the image or description changes on wikipedia then it will fail, I'd like to avoid that by intercepting the call to get the real data that is going to be shown

Reply:
cc @hueitan and @stephanebisson

Reply:
take this as the example
```
https://commons.wikimedia.org/w/api.php?format=json&formatversion=2&origin=*&action=query&prop=imageinfo&iiextmetadatafilter=License%7CLicenseShortName%7CImageDescription%7CArtist&iiextmetadatalanguage=en&iiextmetadatamultilang=1&iiurlwidth=240&iiurlheight=320&iiprop=url%7Cextmetadata&titles=File%3AWild-domestic-hybrid_cat_skulls.png
```
![image](https://user-images.githubusercontent.com/2560096/129197584-2dd6ccf9-28c0-4913-92b2-e6bc36cd4e7e.png)